### PR TITLE
Add production apps

### PR DESCRIPTION
**Pull request by sitilge**
Add production apps.

### CODE REVIEW

File: production/aggregator/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Create Sealed secret
```
Comment (ausias-armesto) [inline production/aggregator/README.md:1]: I would try to find a better place to place the documentation and not duplicating it.
Alternatives: 
- Put the documentation only in Production
- Put the documentation only in Staging
- Create a directory only for documentation independent of the environment

File: production/aggregator/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+```shell
+cd tmp
+
+export ENVIRONMENT_NAME=production
+
+kubectl create secret generic postgres-certs --namespace ${ENVIRONMENT_NAME} --dry-run=client --type=kubernetes.io/tls --from-file=tls.crt=postgres-production-client-cert.pem --from-file=tls.key=postgres-production-client-key.pem --from-file=ca.crt=postgres-production-ca.pem -o yaml | kubeseal --controller-name=sealed-secrets --controller-namespace=sealed-secrets --format yaml > sealed-secret-postgres-certs.yaml
```
Comment (ausias-armesto) [inline production/aggregator/README.md:8]: Substitute the text `production` with the variable `${ENVIRONMENT_NAME}` so the documentation works for both environments.

File: production/aggregator/application.yaml [hunk 3]
```diff
@@ -15,26 +15,34 @@ spec:
           value: |
             image:
               repository: europe-west6-docker.pkg.dev/ctdapp-391309/ctdapp/cover-traffic
-              tag: "0195a42"
-            replicaCount: 1
-            args: 
+              tag: "2a8d28b"
+            replicaCount: 0
```
Comment (Teebor-Choka) [inline production/aggregator/application.yaml:19]: Disable?

**Review by ausias-armesto — CHANGES_REQUESTED** [review]

File: production/aggregator/application.yaml (hunk 3 above)
Comment (jeandemeusy) [reply production/aggregator/application.yaml:19]: For production yes it should be set to 0 until we have our staging version finalised

File: production/rabbitmq/sealed-secret-ctapp.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    RABBITMQ_VIRTUALHOST: AgC0BzyzV0GwFDPr+hHZfNdUrqN06qQwtJVDEj846L3CY97bnxidKVMA5Xk4IHd6vkMJmnxlI39n1dCP7meVU7ohikBaYyx6zp41Qc9qKMcUqrhs7U7oHjiRkgdSKYkWeB+2b+Z+hZ4yV1AweepOX8A4Kj8CCxyKjAn9v15eI87jW7yvMzcumxwLALxVonkYsSe1LohmTmSIGw3hkH/A1TKuWlNQ4dZfjMjmD3H7VdJX6ovy0BAaXiv+E694/Hx4+wjJZaCEJBCsiAUBjUSl62PDcevhOWfBjlu6eCMr7J2l2y+KrFlBXhmxsCEyHsO14DXwW6vYkVELu67uBvOytxJaKpckk50TPG0pyxUSMZ5MafdRfpmkn23gthjDbnr3ExzYaDxcNB20ufLLwNdt99IylDmQ5HTNCmTWGBY7wagsfMm0l9imQaNj6sXCOK5L8wdgnneUCaBbkVKir5mJYkI1++TInHHc9YdcdfMV3+x/nXmKivVPBy4v0g3cQ8JuMsX1aXZIcnTqVwV08z6K3RlwVPAOT0bEfTBnwIEOVAPMMm54L+2MwvLif6FnQAShBXcnQRXIc6/xYsei3qtdzz7xzM98Q3mraYH2fyhmH+GKKNf9eR50tntKlUP+MflxT0w/p15xv5nIbIYyF+bqSkaW3ygbwz5Lt/hg2gBMsTJI1zptfIsdLaX0ub2jh/U5paR3E6lpoLjCxo9J7VBV/RefrQ==
```
[the remark this answers is not in the document]
Comment (jeandemeusy) [reply production/rabbitmq/sealed-secret-ctapp.yaml:13]: This means that the `RABBITMQ_*` are useless ?

**Review by Teebor-Choka — APPROVED** [review]

**Review by ausias-armesto — APPROVED** [review]

**Pull request merged**